src/Microsoft.DocAsCode.Build.Engine/PostbuildPhaseHandlerWithIncremental.cs

```diff
@@ -88,6 +92,28 @@ private void ReloadModelsPerChanges(IEnumerable<HostService> hostServices)
             }
         }
 
+        private void RegisterUnloadedXRefSpec()
+        {
+            var lastXrefMap = LastBuildVersionInfo?.XRefSpecMap;
+            foreach (var m in _unloadedFiles)
+            {
+                if (lastXrefMap == null)
+                {
+                    throw new BuildCacheException($"Full build hasn't loaded XRefMap.");
+                }
+                IEnumerable<XRefSpec> specs;
+                if (!lastXrefMap.TryGetValue(m, out specs))
```

Comment:
no, if a file doesn't report any uids, it will have a record of empty in BuildVersionInfo.XRefSpecMap

---
In reply to: [90193405](https://github.com/dotnet/docfx/pull/936#discussion_r90193405) [](ancestors = 90193405)